### PR TITLE
chore(matsucon2024): Add additional info field to vendor application form

### PR DESCRIPTION
**Pull request by jkjuopperi**

### CODE REVIEW

File: backend/events/matsucon2024/forms/vendor-application-en.yml [hunk 1]
```diff
@@ -97,6 +97,11 @@ fields:
     type: FileUpload
     required: true
 
+  - slug: additional_info
```
Comment (PastelPasty) [inline backend/events/matsucon2024/forms/vendor-application-en.yml:100]: add helpText: "For any comments or info that didn't fit the categories above"

File: backend/events/matsucon2024/forms/vendor-application-fi.yml [hunk 2]
```diff
@@ -97,6 +97,11 @@ fields:
     type: FileUpload
     required: true
 
+  - slug: additional_info
```
Comment (PastelPasty) [inline backend/events/matsucon2024/forms/vendor-application-fi.yml:100]: add helpText: "Kommenteille ja muulle infolle joka ei sopinut aikaisempiin kenttiin."

**Review by PastelPasty — COMMENTED** [review]
Missing helptext

File: backend/events/matsucon2024/forms/vendor-application-en.yml (hunk 1 above)
Comment (jkjuopperi) [reply backend/events/matsucon2024/forms/vendor-application-en.yml:100]: added

File: backend/events/matsucon2024/forms/vendor-application-fi.yml (hunk 2 above)
Comment (jkjuopperi) [reply backend/events/matsucon2024/forms/vendor-application-fi.yml:100]: added

**Review by PastelPasty — APPROVED** [review]

**Pull request merged**